msal/application.py

```diff
@@ -75,9 +75,10 @@ def _parse_pfx(pfx_path, passphrase_bytes):
     x5c = [
         '\n'.join(cert_pem.splitlines()[1:-1])  # Strip the "--- header ---" and "--- footer ---"
     ]
+    sha256_thumbprint = cert.fingerprint(hashes.SHA256()).hex()  # cryptography 0.7+
```

Comment:
Can you do the below checks in this function so that you're only returning one thumbprint?

Reply:
You mean a test case? We do not currently have such a test case, but its utilization is in [this part of the PR](https://github.com/AzureAD/microsoft-authentication-library-for-python/pull/722/files#diff-4d32d284fa6acf43894719eff11cb366e5969e0b61593bfd00c68a4cf9c441caR772-R781). You can see that we pass in only one of the thumbprints, not both.